starfish/test/full_pipelines/cli/_base_cli_test.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+from typing import List, Tuple
+
+from starfish.intensity_table import IntensityTable
+from starfish.util import clock
+
+
+class CLITest(unittest.TestCase):
+    __test__ = False
+
+    SUBDIRS = Tuple[str]
+    STAGES = Tuple[List]
+
+    def verify_results(self, intensities):
```

Comment:
Instead of pass I'd use `raise NotImplementedError()`

Reply:
Also, consider adding type hints.